diagrams/__init__.py

```diff
@@ -138,7 +140,8 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         self.render()
         # Remove the graphviz file leaving only the image.
-        os.remove(self.filename)
+        if (not self.preserve_graphviz_file):
```

Comment:
The parentheses are not needed.

Reply:
That's my javascript coming out... I'll put it back